src/main/java/com/fp3/haras/view/screens/animals/AnimalsScreen.java

```diff
@@ -203,18 +248,49 @@ private void updateBoxSearch(){
         for (Animal animal : fetchedAnimals) {
             boxSearch.addItem(animal.getName());
         }
-        System.out.println(fetchedAnimals);
+    }
+    
+    private void populateTable() {
+        DefaultTableModel table = (DefaultTableModel) tableProgress.getModel();
+        table.setRowCount(0);
+        
+        fetchTableResults(this.currentPage, this.maxResults);
+        for (Animal a : this.tableResults) {
+            table.addRow(new Object[]{
+                a.getId(),
+                a.getName(),
+                a.getCoat(),
+                a.getCategory(),
+                a.getHasExameAie(),
+                a.getHasExameMormo(),
+                a.getHasGta(),
+            });
+        }
+    }   
+    
+    private void fetchTableResults(int page, int maxResults) {
+        EntityManagerFactory emf = Persistence.createEntityManagerFactory("haras");
+        EntityManager em = emf.createEntityManager();
```

Comment:
Here I had to use EntityManager because our EntityUtils does not support limiting the results for now